app/core/ingestion/utils.py

```diff
@@ -128,12 +128,29 @@ class Result:
     details: str = ""
 
 
+NameSummary = tuple[str, str]
```

Comment:
It may be worth updating this to be a "proper" dataclass, because we'll also need to support deletion marks soon...